ACE/ace/DLL_Manager.cpp

```diff
@@ -203,20 +203,6 @@ ACE_DLL_Handle::open (const ACE_TCHAR *dll_name,
                     }
                   open_mode |= RTLD_MEMBER;
 
-                  if (ACE::debug ())
-                    {
-                      ACE_TString err;
-                      ACELIB_DEBUG ((LM_DEBUG,
-                                  ACE_TEXT ("ACE (%P|%t) DLL_Handle::open ")
-                                  ACE_TEXT ("(\"%s\", 0x%x) -> %s: %s\n"),
-                                  aix_pathname,
-                                  open_mode,
-                                  (this->handle_ != ACE_SHLIB_INVALID_HANDLE
```

Comment:
handle_ is always ACE_SHLIB_INVALID_HANDLE

Reply:
This probably should move below the ACE_OS::dlopen which was on line 220, it prints the result of the dlopen

Reply:
Yes, your are right.

Reply:
Fixed